src/components/Table/index.tsx

```diff
@@ -1,30 +1,49 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import classNames from 'classnames';
 
 import styles from '@/components/Table/Table.css';
 export interface Props {
-  headerList: React.ReactNode[];
-  tableContent: React.ReactNode[][];
   footerList?: React.ReactNode[];
+  headerList: React.ReactNode[] | string[];
   onRowClick?: (row: React.ReactNode, rowIndex: number) => void;
   onRowMouseEnter?: (row: React.ReactNode, rowIndex: number) => void;
   onRowMouseLeave?: (row?: React.ReactNode, rowIndex?: number) => void;
   renderFooter?: () => React.ReactNode;
+  sortIcon?: React.ReactNode | { iconAsc: React.ReactNode; iconDesc: React.ReactNode };
+  setSortState?: (value: {
+    sortField: string | React.ReactNode;
+    sortOrder: 'ASC' | 'DESC';
+  }) => void;
+  tableContent: React.ReactNode[][];
   variablesClassName?: string;
 }
 
 const Table = (props: Props) => {
   const {
-    headerList,
-    tableContent,
     footerList,
+    headerList,
     onRowClick,
     onRowMouseEnter,
     onRowMouseLeave,
     renderFooter,
+    sortIcon,
```

Comment:
hey @viniciusaa would be possible to add the <ArrowUp /> and <ArrowUp /> as the default icons, since we have them already added, would be easier to use the table with icons without adding any icon, wdyt?